homeassistant/components/zwave_js/api.py

```diff
@@ -2110,15 +2111,42 @@ def forward_stats(event: dict) -> None:
     )
 
 
-def _get_node_statistics_dict(statistics: NodeStatistics) -> dict[str, int]:
+def _get_node_statistics_dict(
+    hass: HomeAssistant, statistics: NodeStatistics
+) -> dict[str, int]:
```

Comment:
The return value dict doesn't only have integers as values. There are nested dicts too and None.